packages/nbdime/src/styles/variables.css

```diff
@@ -27,4 +27,10 @@
 
   --jp-merge-either-color1: #aee;
   --jp-merge-either-color2: #cff4f4;
-}
\ No newline at end of file
+  --jp-expand-outer-wrapper: rgb(221, 244, 255, 1);
+  --jp-expand-color: rgb(87, 96, 106, 1);
+  --jp-expand-activate-color: rgba(84, 174, 255, 0.4);
+  --jp-expand-activate-focus-color: rgb(14, 94, 208, 1);
```

Comment:
Could these be re-written to be partially transparent, so that they have a fighting chance of working on both dark an light themes? Alternatively be based on JupyterLab CSS variables so that they might have inherent styling?